mantidproject/fitbenchmarking
# Added scipy fitting

#### Description of Work

Fixes #118 
Scipy algorithm curve_fit is now being benchmarked using all the available minimizers.

#### Testing Instructions

1. In `example_runScripts.py` check if `algorithms = 'scipy'`, this should enable fitting using the scipy algorithms. 
2. In the `mantidpython` terminal, run `example_runScripts.py`
3. Again, in `example_runScripts.py` change `algorithms='scipy'` to `algorithms='mantid'` and check if the script produces the same result as before (once approved the cron job will do the checking for this step more thoroughly).
